website/docs/content/writing-guidelines/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Writing Guidelines
+caption: Guidelines for writing consistent UI copy.
+description: Guidelines for writing consistent UI copy.
+previewImage: assets/illustrations/content/writing-guidelines.jpg
+---
+
+## Date and time
+
+Present dates and times in a clear and consistent format to enhance usability across products.
+
+### Usage
+
+Follow these formatting options to display dates and times in a way that aligns with user needs: 
+- **Relative date**: Displays time in relative terms (e.g., "3 hours ago" or "in 2 days"). This format is recommended for recent events, particularly within the past or upcoming week, as it simplifies recognition of time-sensitive actions.
+- **Friendly date**: Provides a user-friendly date, optionally paired with a relative indicator (e.g., "Sep 5, 2018, 3:30 PM EST"). Ideal for longer time frames where the exact date offers clear context. Limit time format to the minute.
+- **Friendly local date**: Converts the date to the user’s local timezone for added clarity (e.g., "Sep 2, 2024, 3:30 PM EST"). Limit time format to the minute.
```

Comment:
```suggestion
- **Friendly local date**: Converts the date to the user’s local timezone for added clarity, e.g., "Sep 2, 2024, 3:30 PM EST". Limit time format to the minute.
```

When would a consumer want to use a friendly date vs a friendly local date?

Reply:
I think I have an idea of how to write this better.

Reply:
Is the update I made better? What I'm trying to say is like an event is set by a company in California and I'm in EST, so seeing the time set in PDT shows me they are 3 hours behind me.